icubam/predicu/plot/flux_dept_lits_dept_visu_4panels.py

```diff
@@ -0,0 +1,479 @@
+import datetime
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+import icubam
+import icubam.predicu.data
+import icubam.predicu.plot
+
+data_source = "bedcounts"  ## We ALSO need that one, ... but only to count how many ICUs there are per department !
+data_source = "combined_icubam_public"
```

Comment:
We should make `data_source` a list, otherwise it would only take the second value.

Reply:
yep